standards/out-of-hours.md

```diff
@@ -0,0 +1,60 @@
+
+## Determining if your service needs support out of hours
+
+Not all services need out of hours support. When considering the need for out of hours support for your service, you first need to understand the risk if the service were to go down when no-one is around to fix it.
+
+The kinds of risk that might suggest out of hours support is necessary include:
+
+- Risk to someone's life
+- The MOJ's ability to provide critical, time-sensitive justice services
+- Impact to staff's and suppliers' ability to do carry out their job safely and to the best of their ability
+- Impairing citizens', suppliers', and partners' ability to interact with MOJ services in a way that meets their needs
+
+These risks should be considered alongside your service's robustness.
+
+For example, if your service can recover from most classes of issue itself, and downtime means minor impairment to a small group of people whose task is not time-sensitive, out of hours support may not be necessary. If, however, a problem with your service (whether outages, security breaches, or something else) out of hours presents a genuine risk to someone's life, it _may_ be appropriate to provide out of hours support around the clock, depending on the robustness of the system.
+
+## Assessing your team's ability to provide out of hours support
+
+To run a sustainable out of hours rota, you must first ensure that either:
+
+- You have enough staff (including contractors) to support the service out of hours sustainably (i.e. without risking burnout), and have the ability to pay them for their time supporting the service
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call out of hours impacts people in different ways. Some people (especially people new to being on call) find the idea of being on call inherently stressful, and put their lives on hold while they are on call. Others are able to integrate being on call into their lives, happily taking their laptop and phone wherever they go and moderating their activities to allow that.
+
+Either kind of person, and all of those on the spectrum between, however, can easily burn out if a few core criteria are not met:
+
+- No one person is on call more than 50% of the time, and ideally no more than 30% of the time.
+- At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).
```

Comment:
Not sure two on call is desirable, perhaps there's a threshold for team size that must be met before this is achievable (balance against frequency of on call on either level) - for eg team of 8, you're on call either pri/sec one in four.

Reply:
I feel two minimum is fair as a person may need support in a difficult situation and knowing it's possible/ acceptable to contact that other person can reduce stress/ anxiety significantly.

Also, perhaps we need to include a statement on which level of staff is acceptable to be on call, which is possibly where things become tricky. My feeling is that senior and mid level developers can be on call, however, probably not two mid levels. It's probably infeasible to require a senior to be on call. I'm interested to hear thoughts here. I feel a junior can a secondary and should experience on call, but possibly not if a mid-level is primary? I suppose decisions will need to be made pragmatically on a team by team basis, but some guidance will probably be useful.

Reply:
As @Tocacar says, I think two on call at any one time *should* be (but currently isn't) the minimum.

For people who're very used to being on call solo (as you are, @nickb834), that can totally seem overkill. On the other hand, though, it provides much better support for people who're new to being on call, as well as providing a backup if the primary has some sort of emergency they have to deal with (or just can't get to their phone right away).

@Tocacar To your point about which levels of staff can be on call: that's definitely something I'd like to cover here, but I've opened #59 to remind me to add that after I've got the basic guidance in place. I hope that's alright?

Reply:
@nickb834 And, yeah, the implication of this and the prior line (“No one person is on call more than 50% of the time, and ideally no more than 30% of the time.”) is that a team of less that 6 people can't sustainably run an on-call rota without violating one of these two things.